groovy-language-server/src/main/java/com/palantir/ls/server/util/SourceWriter.java

```diff
@@ -210,8 +214,8 @@ public synchronized void saveChanges() throws IOException {
     private synchronized void initialize() throws IOException {
         if (!destination.toFile().exists() && destination.toFile().isDirectory()) {
             if (!destination.toFile().mkdirs()) {
-                throw new RuntimeException(
-                        String.format("Could not recreate destination directories: %s", destination.toString()));
+                logger.error("Could not recreate destination file '{}'", destination.toString());
+                throw new RuntimeException("Could not recreate destination directories");
```

Comment:
add `Make sure you have permission to modify your system's temporary directory` here too